photon-core/src/main/java/org/photonvision/vision/frame/FrameStaticProperties.java

```diff
@@ -104,4 +137,14 @@ public static DoubleCouple calculateHorizontalVerticalFoV(
 
         return new DoubleCouple(Math.toDegrees(horizontalView), Math.toDegrees(verticalView));
     }
+
+    @Override
+    public void release() {
+        for (FrameStaticProperties prop : cachedRotationStaticProperties) {
+            if (prop != null) {
+                prop.release();
+            }
+        }
+        cameraCalibration.release();
```

Comment:
This is causing issues, because the cameraCalibrationCoefficients here is owned by someone else. So releasing it breaks everyone. Needs more thought on the ownership model here. God I hate java